src/test/scala/scalaz/stream/NioSpec.scala

```diff
@@ -203,13 +203,13 @@ class NioSpec extends Properties("nio") {
     Task(
       (clients onComplete eval_(stop.set(true)))
       .runLast.map(v=>v.map(_.toSeq).toSeq)
-      .runAsync(clientGot.put)
-    ).run
+      .unsafePerformAsync(clientGot.put)
+    )unsafePerformSync
 
     clientGot.get(6000)
     serverGot.get(6000)
 
-    stop.set(true).run
+    stop.set(true)unsafePerformSync
```

Comment:
dot please. can you please assures dot are putted everywhere when chaining the calls?